### ISSUE TITLE
overscanColumnCount / overscanRowCount are missing in MultiGrid

### ISSUE THREAD

**Issue by kachkaev** [bvaughn/react-virtualized#547]
After playing with the `Grid` component, I found `MultiGrid` and switched to it in my code. This was a very quick and intuitive transition, which I can't even imagine in a non-React project. What I discovered though was that `overscanColumnCount` and `overscanRowCount` props stopped working and so cells with some heavy graphics started flickering when I scrolled.

According to the source and the docs, `MultiGrid` does not support `overscanColumnCount` and `overscanRowCount` at the moment. Is this done on purpose or just there has not been enough time to implement these props? If it's the second case, I can submit a PR but I'm not sure it'll be perfect.

**Comment by bvaughn** [comment]
Hi. I'm not sure what docs you're referring to, but `MultiGrid` _does_ support `overscanColumnCount` and `overscanRowCount`. For example [see here](https://github.com/bvaughn/react-virtualized/blob/be389f9e07bf660262220639cbb0667b0d1cb212/source/MultiGrid/MultiGrid.js#L436). `MultiGrid` passes through most props as-is to the child `Grid` (including `overscanColumnCount` and `overscanRowCount`). Cheers~

**Comment by kachkaev** [comment]
Thank you for your quick reply and sorry for this false alarm:

Here is what confused me:
- I misunderstood the second paragraph in [MultiGrid.md](https://github.com/bvaughn/react-virtualized/blob/master/docs/MultiGrid.md), thinking that only the properties in a table are meant (I now see that `cellRenderer` is not in the table, but is explicitly mentioned, so my assumption cannot be right). Maybe the table could be made a bit clearer by adding an extra row that contains a link to `Grid`'s props? This could make the idea of such an 'inheritence' easier to grasp.

- I checked the source for `MultiGrid` and did not find `overscanColumnCount` & `overscanRowCount` [in `propTypes`](https://github.com/bvaughn/react-virtualized/blob/8b74fc6c84455c707f6692c1a916cba76c6300d5/source/MultiGrid/MultiGrid.js#L14). How about adding `...Grid.propTypes` here?

- I simply did not spot `{...props}` inside `_render***` methods.

Hope this feedback helps a bit.
